tensorflow/compiler/mlir/lite/transforms/post_quantize.cc

```diff
@@ -97,6 +102,28 @@ void RemoveQuantizationAdaptorOps(FuncOp func) {
     bb.eraseArgument(0);
   }
 
+  int num_return_operands = terminator->getNumOperands();
```

Comment:
You can get the mlir::FunctionType and then query the results directly to get the result types.